### PR TITLE
Remove instances of _q and _u from examples.

### PR DESCRIPTION
**Pull request by moorepants**
Fixes issue #138.

- Minor edit to the three link pendulum.
- Removes mass_spring example, as it was an early hacked demo and does not use
  the current API, plus the mass_spring_damper is a better example that does
  everything it did.

- [x] There are no merge conflicts.
- [x] If there is a related issue, a reference to that issue is in the
  commit message.
- [x] The code follows PEP8 guidelines. (use a linter, e.g.
  [pylint](http://www.pylint.org), to check your code)
- [x] The bug fix is documented in the [Release
  Notes](https://github.com/pydy/pydy#release-notes).
- [x] All reviewer comments have been addressed.

### CODE REVIEW

**Comment by moorepants** [comment]
@pydy/pydy-developers This PR needs a review, thanks!

**Comment by sahilshekhawat** [comment]
looks like there are merge conflicts, can you please fix them?

**Comment by moorepants** [comment]
Merge conflicts fixed.

File: examples/three_link_conical_pendulum/simulate.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from pydy.codegen.code import generate_ode_function
```
Comment (sahilshekhawat) [inline examples/three_link_conical_pendulum/simulate.py]: Shouldn't we migrate our examples to new ``generate_ode_function``?

**Comment by sahilshekhawat** [comment]
These changes does fix ``TypeError: <lambda>() takes exactly 25 arguments (31 given)`` which occurs in master but animation still doesn't work. Is it a known issue?

File: examples/three_link_conical_pendulum/simulate.py (hunk 1 above)
Comment (moorepants) [inline examples/three_link_conical_pendulum/simulate.py]: I can fix that one here. Should be a quick fix.

**Comment by moorepants** [comment]
The animation browser works for me. This example works perfectly on my machine from derivation to visualization. Can you specify what isn't working for you?

**Comment by moorepants** [comment]
> These changes does fix TypeError: <lambda>() takes exactly 25 arguments (31 given) which comes in master

Yes this is the error this is trying to fix.

**Comment by sahilshekhawat** [comment]
It is working fine now, animation was not working before. Thanks for this fix :)

**Pull request merged**